### PR TITLE
doc-3314 Changed repo from Maven Central to Artifactory for Tokens SDK

### PR DESCRIPTION
**Pull request by michelemarsden**

### CODE REVIEW

File: content/en/platform/corda/5.0-dev-preview-1/tokens-sdk/overview.md [hunk 1]
```diff
@@ -36,16 +36,24 @@ This version of the Tokens SDK is for the Corda 5 Developer Preview only. It can
 
 ## Using the Tokens SDK
 
-The binaries for the Tokens SDK version `2.0.0-DevPreview-1.0`, which is compatible for use with the Corda 5 Developer Preview, 
-are released on Maven Central.
+The binaries for the Tokens SDK version `2.0.0-DevPreview-1.0`, which is compatible for use with the Corda 5 Developer Preview,
+are released on the corda-lib Artifactory.
 
 If your project is set up to import dependencies from Maven Central, you can pull in these binaries.
```
Comment (blsemo) [inline content/en/platform/corda/5.0-dev-preview-1/tokens-sdk/overview.md:42]: This line needs to be deleted - must have forgotten that.

**Review by blsemo — COMMENTED** [review]
Looks good apart from the one line mentioning maven central.

**Review by blsemo — APPROVED** [review]
LGTM

**Review by ivanterziev-r3 — APPROVED** [review]
LGTM

**Pull request merged**